### PR TITLE
Add c_position to variant report

### PR DESCRIPTION
**Pull request by dufeiyu**
c_position column from vep output need to be added to report. This PR also replaces the obsolete CGI with URI::Escape, which fixes a bug that unescapes + into space with CGI.

### CODE REVIEW

**Comment by tmooney** [comment]
+1 :fallen_leaf:

File: lib/perl/Genome/File/Vcf/VepConsequenceParser.pm [hunk 1]
```diff
@@ -5,7 +5,7 @@ package Genome::File::Vcf::VepConsequenceParser;
 
 use Data::Dumper;
 use Carp qw/confess/;
-use CGI qw/unescape/;
+use URI::Escape;
```
Comment (brummett) [inline lib/perl/Genome/File/Vcf/VepConsequenceParser.pm]: Even though it automatically exports ```uri_unescape()```, it's still good practice to put it in the import list
```
use URI::Escape qw(uri_unescape);
```

Did you figure out whether this data is sourced from an actual URL and whether uri_unescape is the right thing to do here?

Comment (dufeiyu) [inline lib/perl/Genome/File/Vcf/VepConsequenceParser.pm]: ok, I will add to import list.

**Comment by brummett** [comment]
+1

**Pull request merged**